src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionState.ts

```diff
@@ -1,27 +1,37 @@
-import { CustomDownloadAttributes } from 'src/content/app/custom-download/types/Attributes';
+import Attributes, {
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/62013322-9a113b80-b188-11e9-8d3a-d53f5154b1fa.png)

Reply:
Removed it